nodejs/package.json

```diff
@@ -17,7 +17,7 @@
   },
   "homepage": "https://github.com/mongodb-university/csfle-guides#readme",
   "dependencies": {
-    "mongodb": "mongodb/node-mongodb-native#master",
+    "mongodb": "^3.4.0",
```

Comment:
Looks like `mongo-client-encryption` was updated to 1.0.0 yesterday with the GA release: https://www.npmjs.com/package/mongodb-client-encryption

Reply:
Thanks, updated this as well